packages/account/src/Components/api-token/api-token-table.tsx

```diff
@@ -104,14 +107,15 @@ const ApiTokenTable = () => {
         );
     }
 
+    const api_token_table_header_title = ['Name', 'Token', 'Scopes', 'Last used'];
+
     return (
         <table className='da-api-token__table'>
             <thead>
                 <tr className='da-api-token__table-header-row'>
-                    <ApiTokenTableRowHeader text={localize('Name')} />
-                    <ApiTokenTableRowHeader text={localize('Token')} />
-                    <ApiTokenTableRowHeader text={localize('Scopes')} />
-                    <ApiTokenTableRowHeader text={localize('Last used')} />
+                    {api_token_table_header_title.map(title => (
+                        <ApiTokenTableRowHeader key={title} text={<Localize i18n_default_text={title} />} />
```

Comment:
`Localize` will work if the translation is already there for these strings, however if there's a new word which is not translated yet, it'll not be sent for translation if we're using a variable

Reply:
lets use localize for now